scripts/run-framework/agief_experiment/experiment.py

```diff
@@ -192,16 +192,33 @@ def run_parameterset(self, compute_node, cloud, args, entity_filepath, data_file
             # log results expressed in the appropriate entity config
             self.log_results_config(compute_node)
 
+            # Get the path to labels and features CSV files
+            out_labels_file_path = self.experiment_utils.datapath('labels.csv')
+            out_features_file_path = self.experiment_utils.datapath('features.csv')
+
             if args.export:
                 out_entity_file_path, out_data_file_path = self.experiment_utils.output_names_from_input_names(
```

Comment:
Is there a flag to toggle this export off, since we don't need it for this new approach?

Likewise for `export_compute`.